### PR TITLE
Fix system dateTime loading in avalonia LoadCurrentConfiguration

### PR DESCRIPTION
**Pull request by nico-abram**
Fixes https://github.com/Ryujinx/Ryujinx/issues/6632

In SettingsViewModel.cs , this line stores the difference in seconds between the current system time and the configured time
`config.System.SystemTimeOffset.Value = Convert.ToInt64((CurrentDate.ToUnixTimeSeconds() + CurrentTime.TotalSeconds) - DateTimeOffset.Now.ToUnixTimeSeconds());`
That in and of itself sounds fine to me, when we go backwards in time it results in a large negative number
Then it loads that value here
```cs
            DateTime currentDateTime = DateTime.Now;

            CurrentDate = currentDateTime.Date;
            CurrentTime = currentDateTime.TimeOfDay.Add(TimeSpan.FromSeconds(config.System.SystemTimeOffset));
```
The date is always set to the current system time date, and then the time gets the entirety of the offset. This can result in a hilariously large negative time-within-the-day that crashes Avalonia's datetime picker

The fix in this PR is just applying the offset to a DateTime instead of just the Time within a day and then getting the Date and Time from the result.

### CODE REVIEW

**Review by TSRBerry — APPROVED** [review]
Lgtm, thank you!

**Review by IsaacMarovitz — APPROVED** [review]

File: src/Ryujinx/UI/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -412,10 +412,11 @@ public void LoadCurrentConfiguration()
             Language = (int)config.System.Language.Value;
             TimeZone = config.System.TimeZone;
 
-            DateTime currentDateTime = DateTime.Now;
-
+            DateTime currentHostDateTime = DateTime.Now;
+            TimeSpan SystemDateTimeOffset = TimeSpan.FromSeconds(config.System.SystemTimeOffset);
```
Comment (riperiperi) [inline src/Ryujinx/UI/ViewModels/SettingsViewModel.cs:416]: This should be called `systemDateTimeOffset`, we only use upper camel case for class/struct, public/internal members and constants.

Comment (nico-abram) [reply src/Ryujinx/UI/ViewModels/SettingsViewModel.cs]: Renamed, thanks! Sorry I missed it, I think I probably copied the field's name.

**Review by riperiperi — APPROVED** [review]
Nice catch.

**Pull request merged**